lib/system/sysio.nim

```diff
@@ -272,17 +277,34 @@ const
     # should not be translated.
 
 when defined(posix) and not defined(nimscript):
-  type
-    Mode {.importc: "mode_t", header: "<sys/types.h>".} = cint
+  when defined(linux) and defined(amd64):
```

Comment:
`glibc/bits/stat.h` is full of `#ifdef __x86_64__` - this is the one I could test..